app/assets/stylesheets/kaui/kaui.less

```diff
@@ -1,3 +1,5 @@
+@import "twitter/bootstrap/bootstrap";
```

Comment:
Is that necessary? It should already be imported (see https://github.com/killbill/killbill-admin-ui/blob/c4fb6993d99c736ffde779f26353eaff60d256a6/app/assets/stylesheets/application.css#L9).